swift/WorkflowUI/Sources/Screen/SubscriptionWorkflow.swift

```diff
@@ -0,0 +1,120 @@
+import Foundation
+import ReactiveSwift
+import Workflow
+
+public struct SubscribedScreen<S: Screen, SubPro: SubscriptionProvider>: Screen {
```

Comment:
How does this interact with the nested `Screen`'s view controller? It gets wrapped by `ViewController` below, right?

Is that going to cause issues with any of our containers?

Reply:
Not sure how. The big negative is we'll be adding a pretty heft VC just to subscribe.